test/v2/connectors/AmendUkPropertyPeriodSummaryConnectorSpec.scala

```diff
@@ -87,34 +86,59 @@ class AmendUkPropertyPeriodSummaryConnectorSpec extends ConnectorSpec {
       ))
   )
 
-  private val request: AmendUkPropertyPeriodSummaryRequest = AmendUkPropertyPeriodSummaryRequest(
-    nino = Nino(nino),
-    taxYear = TaxYear.fromMtd(taxYear),
-    businessId = businessId,
-    submissionId = submissionId,
-    body = requestBody
-  )
-
   trait Test {
```

Comment:
Could Test & requestBody be stuffed down into the end of the test class?  Similar to RetrieveForeignPropertyPeriodSummaryConnectorSpec  - makes the tests easier to read